src/components/install_assistant/InstallAssistantStep10.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+	<InstallAssistantStepTemplate
+		title="6. Abgeschlossen"
```

Comment:
```suggestion
		title="Abgeschlossen"
```
Da diese Seite von beiden Zweigen (primary, secondary) erreicht werden kann, würde ich die Nummerierung hier weglassen. Wäre dann auch analog zu "Start" im Schritt 0.